lib/inspect_task.rb

```diff
@@ -79,6 +79,9 @@ def build_description(store, name, system, scopes, filter, options)
     scopes.each do |scope|
       inspector = Inspector.for(scope).new(system, description)
       Machinery::Ui.puts "Inspecting #{Machinery::Ui.internal_scope_list_to_string(inspector.scope)}..."
+      if scope == "patterns"
```

Comment:
This message would be shown every time even when you inspect an rpm based system.